account_check_payee/wizard/account_payment_register.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2021 Ecosoft Co., Ltd. (http://ecosoft.co.th)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class AccountPaymentRegister(models.TransientModel):
+    _inherit = "account.payment.register"
+
+    payment_method_code = fields.Char(related="payment_method_id.code")
+    check_payee = fields.Char(
+        string="Payee",
```

Comment:
```suggestion
        string="Payee",
        default="-",
```
If no default, and given some account.payment already exists. You will get this error on install.
```
2021-09-19 16:16:41,020 34 ERROR devel odoo.schema: Table 'account_payment_register': unable to set NOT NULL on column 'check_payee' 
```

Reply:
I add a hook for the assigned payee on old documents.